business_requirement_activity_report/README.rst

```diff
@@ -2,9 +2,9 @@
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
 
-=========================
-Timesheet Activity Report
-=========================
+====================================
+Business Requirement Activity Report
```

Comment:
@noahzaozao this module is very specific, it depends on way too many modules to be included in the OCA BR repo. Definitely belongs to odoo-addons